### ISSUE TITLE
User Management Settings

### ISSUE THREAD

**Issue by jojogrosso** [tidusjar/Ombi#954]
**Ombi Version:**

V 2.0.1234

**Update Branch:**

Stable

**Operating System:**

Windows 10 64bit (Shows as 	Windows 8/Windows Server 2012)

**Mono Version (only if your not on windows)**
n/a

**Applicable Logs (from `/logs/` directory or the Admin page):**

```

Logs are currently set to "Error", If i need to provide logs, what should I set it to?

```

**Problem Description:**

The "default" user management settings for the users under "settings" don't seem to be applying to the users.

**Reproduction Steps:**

Basically, I set the permissions there, and when I go to manage the users, they don't have those permissions.

**Comment by tidusjar** [comment]
I think there is a misunderstanding to what that setting does, probably my fault as it's not really explained.

So that setting is there for when you add a new user into your Plex Friends, Ombi will detect that the user is added then apply those settings to the users.

**Comment by jojogrosso** [comment]
Ohh, alright, thanks for the explanation. Is there a way to set default user settings for all users so we don't have to go through each user to enable/disable certain features we want everyone to have?

**Comment by tidusjar** [comment]
@jojogrosso https://github.com/tidusjar/Ombi/issues/957
